mpeg-encoder.js

```diff
@@ -0,0 +1,58 @@
+module.exports = function () {
+  // eslint-disable-next-line
+  importScripts('https://cdnjs.cloudflare.com/ajax/libs/lamejs/1.2.0/lame.min.js')
+
+  // blob.arrayBuffer() polyfill
+  function arrayBufferPolyfill () {
+    return new Promise(function (resolve) {
+      var fr = new FileReader()
+      fr.onload = function () {
+        resolve(fr.result)
+      }
+      // eslint-disable-next-line
+      fr.readAsArrayBuffer(this);
+      // eslint-disable-next-line
+    }.bind(this))
+  }
+  Blob.prototype.arrayBuffer = Blob.prototype.arrayBuffer || arrayBufferPolyfill
```

Comment:
It is bad practice to modify the standard library.

If we need `ArrayBuffer` polyfill just for IE, we can ask developers to add it manually. Let’s avoid unnecessary polyfills for rarely used browsers.

Reply:
safari needs it as well and it's only few lines.

Reply:
OK, let’s use:

```js
var blobToArray
if (Blob.prototype.arrayBuffer) {
  blobToArray = function (blob) {
    return Blob.prototype.arrayBuffer
  }
} else {
  blobToArray = function (blob) {
    // polyfill
  }
}
```

In this case, we will not change the browser system classes

Reply:
done